AutomatedTesting/Gem/PythonTests/editor/test_Editor_Main_Optimized.py

```diff
@@ -0,0 +1,40 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+import pytest
+
+import ly_test_tools.environment.file_system as file_system
+from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorSharedTest, EditorParallelTest, EditorTestSuite
+
+
+@pytest.mark.SUITE_main
```

Comment:
As per the other PR, please add here an xfail until we are sure the optimized tests work fine in jenkins

Reply:
Done.